frontend/src/components/Tables/SitesEnrollmentTable.js

```diff
@@ -55,6 +64,34 @@ export const SitesEnrollmentTable = props => {
                 // pageSizeOptions: [15, 25, 50, 100, 200],
                 exportFileName: `${ title }__${ now.toISOString() }`,
             }}
+
         />
+        </div>
     )
 }
+
+export const RemoveBtn = row => {
+    return(
+        <IconButton variant="outlined"
+                    color="secondary"
+                    title="remove"
+                    onClick={function () {
+                        if (window.confirm("Do you really want to remove this item?")) {
+                            let siteId = row.id
+                            if(siteId === undefined)
+                                siteId = row.siteId
+
+                            let siteName = row.name
+                            if (siteName === undefined)
+                                siteName = row.siteName
+
+                            axios.delete(api.sites, {data: {siteId: siteId, ctsaId: row.ctsaId, siteName: siteName}})
+                                .then(data => window.alert("Item has been removed. Refresh the page to see the change."))
```

Comment:
Sure, that's how it ideally should be. However It is a rare use case and it is propbably not worth to polish now, since there will be larger changes in this area soon.

Reply:
ideally we'd trigger a rerender on just the table and the user would see the updated data.

however, as it currently stands, all the data is fetched on initial load, so the data-fetching and state management would need some refactoring to improve that behavior.

@vladimir2217 perhaps the FlashMessage component would be better than alert, though.